src/Build.UnitTests/Microsoft.Build.Engine.UnitTests.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Hyper-Nit: Is this a BOM change from a text editor?

Reply:
Yep seems like it... Though I thought I only opened it in VS so it's odd but it should be fixed now 👍